examples/bitmap_font_simpletest.py

```diff
@@ -36,4 +36,4 @@
         else:
             pixels = ""
         print("".join(pixels) + " " * (glyph.shift_x - len(pixels)), end="")
-    print()
+    print(end="\r")
```

Comment:
What does this do now? I expected you to remove this print all together.

Reply:
Removing the print statemnet will produce the following
![image](https://user-images.githubusercontent.com/34255413/223825215-cceac240-8df7-4eec-bdab-a52488a54bc4.png)

Using `print(end="\r")` 
![image](https://user-images.githubusercontent.com/34255413/223825362-73318361-f3aa-4b4e-8d55-1427f83f5055.png)


Before with print() there was a blank line for every printed line. I can only capture the last part but as the repl scrolls up, but to give you an idea
![image](https://user-images.githubusercontent.com/34255413/223826326-e464b7d2-0f8d-41ac-a154-f097627910cb.png)

Reply:
The black background screenshots are from a pyportal titano, the last one in grey is from a tio terminal, but it looks the same in the pyportal, I could try to capture it, but you will only see the bottom part of the message as it will scroll up.